tailwind.config.js

```diff
@@ -0,0 +1,15 @@
+// ts-check
+/** @type {import('tailwindcss').Config} */
+
+const kzPresets = require("@kaizen/tailwind").TailwindPreset
```

Comment:
I wonder if naming this `TailwindPreset` is now redundant? Thoughts on calling it `Preset`?

Reply:
True, now it's in its own package probs doesn't need to be explicitly called out.